src/pyedb/dotnet/edb_core/cell/hierarchy/component.py

```diff
@@ -395,12 +399,7 @@ def value(self):
         """
         if self.model_type == "RLC":
             if not self._pin_pairs:
-                if self.type == "Inductor":
-                    return 1e-9
-                elif self.type == "Resistor":
-                    return 1e6
-                else:
-                    return 1
+                return
```

Comment:
Can you justify (in the PR) this change so that we know why previous default behavior is changed ?